### PR TITLE
[OpenVINO-EP] Adding OpenVINO-EP samples to Msft Repo

### PR DESCRIPTION
**Pull request by MaajidKhan**
**Description**: Adding OpenVINO-EP samples

**Motivation and Context**
-> Adding OpenVINO-EP samples for different flavors like CPP, python and c#.
-> Providing Tutorials of how to use the same.

This will help us to showcase power of ONNXRT-OpenVINO-EP to our customers and developers.
They can just run through these samples directly themselves instead of figuring out how to develop samples from scratch.

### CODE REVIEW

**Comment by MaajidKhan** [comment]
Hello @jywu-msft @oliviajain.
I have also created a subsequent PR for updating the documentation for the same.
https://github.com/microsoft/onnxruntime/pull/7827

The samples and documentation are updated w.r.t how we discussed it to be over email.

File: samples/python/OpenVINO_EP_samples/cleanroom_worker_safety/cleanroom_sample_OV_EP.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  scoreMatExp = np.exp(np.asarray(x))
+  return scoreMatExp / scoreMatExp.sum(0)
+
+# color look up table for 5 different classes for cleanroom sample use case
+clut = [(0,0,0),(255,0,0),(255,0,255),(0,0,255),(0,200,0)]
+
+# 5 labels that the tiny-yolov2 model can do the object_detection on
+label = ["unprotected","bunny suit","glasses","head","robot"]
+
+model_file_path = "tiny_yolo_v2_cleanroom.onnx"
```
Comment (oliviajain) [inline samples/python/OpenVINO_EP_samples/cleanroom_worker_safety/cleanroom_sample_OV_EP.py:25]: For all the examples, if the model/ input data for the examples aren't included in the sample folders with the proper name, maybe change the value to TODO: Input with your x path so the user knows they need to add their input/model paths.

**Review by oliviajain — COMMENTED** [review]
Reviewed python samples, need to review csharp/cpp samples

File: samples/python/OpenVINO_EP_samples/cleanroom_worker_safety/cleanroom_sample_OV_EP.py (hunk 1 above)
Comment (MaajidKhan) [reply samples/python/OpenVINO_EP_samples/cleanroom_worker_safety/cleanroom_sample_OV_EP.py]: @oliviajain
for the cleanroom python sample, the users have to use the pre-trained .onnx model provided by us. However when I am trying to push it, it is not getting reflected in Msft/onnxruntime repo. so just created a tar file of the .onnx file and pushed it. The users have to use the same video files as provided in the data folder to test this sample. I have added TODO as requested and a clear documentation is already available for the same.

Yes for the other samples, the users have to download the model from ONNX model zoo and have to use their own data to test the sample. I have added TODO as requested. Also we have a clear documentation for all the samples which would tell them that they have to download the models from onnx model zoo and have to use their own data.

**Comment by oliviajain** [comment]
Synced offline to validate cpp sample on Linux

**Review by oliviajain — APPROVED** [review]

**Comment by jywu-msft** [comment]
can the media files and models be hosted elsewhere and we link to them in our docs? (are we allowed to check in those cleanroom video files?)
we'd like to avoid checking in large binaries into the code repo.

**Comment by MaajidKhan** [comment]
> can the media files and models be hosted elsewhere and we link to them in our docs? (are we allowed to check in those cleanroom video files?)
> we'd like to avoid checking in large binaries into the code repo.

Yes George, you are allowed to check in those cleanroom video files. we got them approved for releasing them outside.
we are just hosting the model for the clean room sample as it's a pre-trained model and can't be found outside.

yes, I agree with your point. These media files are kinda bulky. so, for now, we are really unsure where to host these samples and make it available. so I will go ahead and remove the clean room sample completely for now. Anyways we have other python sample to showcase python API in OpenVINO-EP.

so for all other samples, we are not hosting any media files or models. so we are good.

**Comment by MaajidKhan** [comment]
@oliviajain  @jywu-msft I will be creating a follow up PR for the documentation as well to remove the clean room sample references.

**Review by oliviajain — APPROVED** [review]

**Review by jywu-msft — APPROVED** [review]

**Review by thiagocrepaldi — APPROVED** [review]

**Pull request merged**